recipes/sdbus-cpp/all/conanfile.py

```diff
@@ -0,0 +1,129 @@
+                    self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
+
+    def build_requirements(self):
+        self.build_requires("pkgconf/1.7.3")
+        if self.options.with_code_gen:
+            self.build_requires("expat/2.2.10")
+
+    def requirements(self):
+        self.requires("libsystemd/247.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        src_folder = "{}-{}".format(self.name, self.version)
+        os.rename(src_folder, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_CODE_GEN"] = self.options.with_code_gen
+        self._cmake.definitions["BUILD_DOC"] = False
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_LIBSYSTEMD"] = False
+        self._cmake.configure()
+        return self._cmake
+
+    def fix_tools_cmake(self):
+        cmake_file = os.path.join(self._source_subfolder,
+                                  "tools", "CMakeLists.txt")
+        tools.replace_in_file(
+            cmake_file,
+            "target_link_libraries (sdbus-c++-xml2cpp ${EXPAT_LIBRARIES})",
+            """
+            target_link_libraries (sdbus-c++-xml2cpp ${EXPAT_LIBRARIES})
+            target_include_directories(sdbus-c++-xml2cpp PRIVATE
+                ${EXPAT_INCLUDE_DIRS})
+            """)
+
+    def build(self):
+        if self.options.with_code_gen:
+            # FIXME: remove after
+            #  https://github.com/Kistler-Group/sdbus-cpp/pull/136
+            self.fix_tools_cmake()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "SDBusCpp"
+        self.cpp_info.names["cmake_find_package_multi"] = "SDBusCpp"
+        self.cpp_info.filenames["cmake_find_package"] = "sdbus-c++"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "sdbus-c++"
+        self.cpp_info.components["sdbus-c++"].libs = ["sdbus-c++"]
+        self.cpp_info.components["sdbus-c++"].requires.append("libsystemd::libsystemd")
+        self.cpp_info.components["sdbus-c++"].names["cmake_find_package"] = "sdbus-c++"
+        self.cpp_info.components["sdbus-c++"].names["cmake_find_package_multi"] = "sdbus-c++"
+        self.cpp_info.components["sdbus-c++"].names["pkg_config"] = "sdbus-c++"
+        if self.options.with_code_gen:
+            bin_path = os.path.join(self.package_folder, "bin")
```

Comment:
```suggestion
            bin_path = os.path.join(self.package_folder, "bin")
            self.output.info("Appending PATH env var with : {}".format(bin_path))
```

That's a common pattern when adding something to PATH.